docs/source/receptorctl/receptorctl_status.rst

```diff
@@ -0,0 +1,171 @@
+------
+status
+------
+
+.. contents::
+   :local:
+
+``receptorctl status`` displays the status of the Receptor network.
+
+The command syntax is: ``receptorctl --socket=<socket_path> status [--json]``
+
+``socket_path`` is the control socket address to connect to Receptor.
+   It defaults to Unix socket.
+   Use tcp:// for TCP socket.
+   The corresponding environment variable is ``RECEPTORCTL_SOCKET``
+
+``ss --listening --processes --unix 'src = unix:<socket_path>'``
+Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <<pid>``
+
+``--json`` will format the output as JSON.
+   The default is human-readable.
+   Using this argument allows output to be machine consumable.  For example, piping into ``jq``
+
+The output is divided into sections listed below.  Field values may be listed in their own section.
+Columns are the actual JSON node values.
+
+^^^^^^^^^^^^
+Node section
+^^^^^^^^^^^^
+
+.. list-table:: Node section
+      :header-rows: 1
+      :widths: auto
+
+      * - Column
+        - Description
+      * - ``."NodeID"``
+        - Node Identifier.
+      * - ``."SystemCPUCount"``
+        - Number of logical CPU cores on the node.
+      * - ``.SystemMemoryMiB"``
+        - Available Mmemory (MiB) of the node.
+      * - ``."Version"``
+        - Receptor version.
+
+^^^^^^^^^^^^^^^^^^^
+Connections section
+^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: Connections section
+    :header-rows: 1
+    :widths: auto
+
+    * - Column
+      - Description
+    * - ``."Connections"``
+      - Connections.
+    * - ``."Connections"."Cost"``
+      - Metric (route preference) to reach NodeID.
+    * - ``."Connections"."NodeID"``
+      - Node ID.
```

Comment:
Is this any different to the NodeID in the Node section?

Reply:
Deferred this discussion